.github/workflows/deploy-branch.yml

```diff
@@ -0,0 +1,21 @@
+name: Docker Branch Deploy
+
+on:
+  pull_request_review:
+    types: [submitted]
```

Comment:
Nit - I think it makes sense to publish with new commits too.  If changes are requested, I doubt we wanna wait for another review to deploy them

Reply:
I looked at that event as well, but we need to make sure that only core developers can trigger a release, so I had to base the workflow on something only core developers can do. Approval seemed like the best option, but I'm open to ideas.

Reply:
I tested a few different methods, including `pull_request.author_association`, but it looks like the best way to make sure only dev can deploy will be to limit actions to this repo (turn them off for forks) and then apply branch rules as needed.